test/integration/osl_mysql_test/controls/osl_mysql_test.rb

```diff
@@ -0,0 +1,11 @@
+# Ensure that the port is bound, and the service is running.
+
+describe service('mysql.service') do
+  it { should be_enabled }
+  it { should be_running }
+end
+
+describe port(3306) do
+  it { should be_listening }
+  its('processes') { should cmp 'mysqld' }
+end
```

Comment:
Please add tests for ensuring the databases exist using the [mysql_session resource](https://docs.chef.io/inspec/resources/mysql_session/).